packages/config-array/src/config-array.js

```diff
@@ -1010,18 +1010,33 @@ export class ConfigArray extends Array {
 			 * a file with a specific extensions such as *.js.
 			 */
 
-			const universalFiles = config.files.filter(pattern =>
-				universalPattern.test(pattern),
-			);
+			const nonUniversalFiles = [];
+			const universalFiles = config.files.filter(element => {
+				if (Array.isArray(element)) {
+					if (
```

Comment:
Maybe we can add a comment here to explain that because an array is an AND operation, then all items in the array need to be universal for it to be considered universal? (Assuming I'm reading this correctly.)

Reply:
Added a comment in https://github.com/eslint/rewrite/pull/218/commits/e5d32707e5d5c7c6b813531a9946fee55b447573